sim/verilator.wake

```diff
@@ -64,51 +66,75 @@ def compileCmdlineArgs options =
   ++ main
 
 def compileInputs options =
-  def outputDir    = options.getVerilatorCompilePlanOutputDir
-  def includeDirs  = options.getVerilatorCompilePlanIncludeDirs
-  def srcFiles     = options.getVerilatorCompilePlanSourceFiles
-
+  def outputDir = options.getVerilatorCompilePlanOutputDir
+  def includeDirs = options.getVerilatorCompilePlanIncludeDirs
+  def srcFiles = options.getVerilatorCompilePlanSourceFiles
+  def visible = options.getVerilatorCompilePlanVisible
```

Comment:
I figure I might as well ask this code style question here: What are your thoughts on vertically aligning assignment statements? I see here that you have switched away from vertically aligning the `=` signs and instead to just having a single space on both sides of the `=` sign. My personal preference is to not vertically align assignment statements, since I feel that too often you will add a variable with a longer name and either have to touch all the other lines or not have everything line up anymore. I'm fine with either approach, but I'd rather that we all have a consistent style instead of changing back and forth.

Reply:
yeah I used to like aligning things but now I feel like it's more trouble than it's worth so I'm switching things back.